fake_switches/netconf/capabilities.py

```diff
@@ -61,7 +61,7 @@ def edit_config(self, request):
 
         return Response(etree.Element("ok"))
 
-    def commit(self, _):
+    def commit_configuration(self, _):
```

Comment:
as other versionsof ncclient worked in prod, i think we should leave the commit()  also so that it works with older versions

Reply:
Ok, will fix